src/hooks/mongodb.js

```diff
@@ -73,20 +73,26 @@ const onFailedHook = (event) => {
 };
 
 export const hookMongoDb = (mongoLib) => {
-  const mongoClient = mongoLib || safeRequire('mongodb');
-  if (mongoClient) {
-    logger.info('Starting to instrument MongoDB');
-    const listener = mongoClient.instrument({}, (err) => {
-      if (err) logger.warn('MongoDB instrumentation failed ', err);
+  const mongoClient = mongoLib ? mongoLib : safeRequire('mongodb');
+  const mongoosClients = safeRequire('node_modules/mongoose/node_modules/mongodb');
+  const mongoClients = [mongoClient, mongoosClients].filter(Boolean);
+  if (mongoClients.length > 0) {
+    mongoClients.forEach((mongoClient) => {
```

Comment:
you don't need the `if` when you're doing `forEach` 🤔

Reply:
this is for the log in the else